sdk/communication/azure-communication-rooms/swagger/README.md

```diff
@@ -0,0 +1,56 @@
+# Azure Communication Service room client library for Java
+
+> see https://aka.ms/autorest
+## Getting Started
+
+To build the SDK for Rooms Client, simply Install AutoRest and in this folder, run:
+
+### Setup
+```ps
+Fork and clone https://github.com/Azure/autorest.java
+git checkout main
+git submodule update --init --recursive
+mvn package -Dlocal
+npm install
+npm install -g autorest
+```
+
+### Generation
+
+There is one swagger for Rooms management APIs.
+
+```ps
+cd <swagger-folder>
+autorest README.md --java --v4 --use=@autorest/java@4.0.59
+```
+
+## Update generated files for rooms service
+To update generated files for rooms service, run the following command
+
+> autorest README.md --java --v4 --use=@autorest/java@4.0.59
```

Comment:
You can add all these params inside the `yaml` section below. Also, `--v4` is not required.

```yaml
use: '@autorest/java@4.0.59'
java: true
```

Reply:
Addressed